janusgraph-berkeleyje/src/main/java/org/janusgraph/diskstorage/berkeleyje/BerkeleyJEStoreManager.java

```diff
@@ -142,6 +143,13 @@ public BerkeleyJETx beginTransaction(final BaseTransactionConfig txCfg) throws B
                 TransactionConfig txnConfig = new TransactionConfig();
                 ConfigOption.getEnumValue(effectiveCfg.get(ISOLATION_LEVEL),IsolationLevel.class).configure(txnConfig);
                 tx = environment.beginTransaction(null, txnConfig);
+            } else {
+                if (txCfg instanceof TransactionConfiguration) {
+                    if (!((TransactionConfiguration) txCfg).isSingleThreaded()) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://app.codacy.com/app/JanusGraph/janusgraph/pullRequest?prid=4527083)